modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -68,17 +106,13 @@ public function add_iframed_body_class( $classes ) {
 	 * @return bool
 	 */
 	public function framing_allowed() {
-		if ( empty( $_GET['frame-nonce'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
-			return false;
-		}
-
 		$verified = $this->verify_frame_nonce(
 			$_GET['frame-nonce'],  // phpcs:ignore WordPress.Security.NonceVerification
 			'frame-' . Jetpack_Options::get_option( 'id' )
 		);
 
 		if ( is_wp_error( $verified ) ) {
-			wp_die( $verified );
+			wp_die( esc_html( $verified ) );
```

Comment:
since we are changing this, how about adding an argument to specify a custom response code to replace the default `500`?

Reply:
Resolved in https://github.com/Automattic/jetpack/pull/11354